### ISSUE TITLE
Is the example complete for Android?

### ISSUE THREAD

**Issue by josiahwitheford** [dev-event/react-native-maps-draw#2]
Hi there, i am doing some investigations into whether this library is going to work for our needs. I have copied the example code into our app and the map shows, however I can't seem to draw any polygons.

I also noticed that if `modePolygon` is set to false then I can't scroll around the map, or click on any other components on the screen, other than the draw on/off button.
I am just wondering if the example is complete for Android?

Cheers

Here's some more information in case it helps

```react-native: 0.65.0
react: 17.0.2
react-native-gesture-handler: 1.10.3
react-native-svg: 12.1.1
react-native-maps-draw: 0.1.4

Android Version: 11
Pixel 5, API 30 -- Emulated
```

**Comment by dev-event** [comment]
@josiahwitheford hi) 
I will test and fix the error

**Comment by dev-event** [comment]
@josiahwitheford Fixed in version 0.1.5
